spec/system/admin/reservation_actions_spec.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
+RSpec.describe "Reservation actions", :js, feature_setting: { cross_core_projects: true } do
+  include_context "cross core orders"
+
+  let!(:reservation) { create(:purchased_reservation, order_detail: cross_core_reservation_order.order_details.first) }
+
+  before do
+    allow_any_instance_of(InstrumentIssue).to receive(:send_notification).and_return(true)
+
+    login_as facility_administrator
+    visit facility_order_path(facility, originating_order_facility1)
+  end
+
+  describe "Report an Issue" do
+    it "redirects to original order show" do
+      find("h3", text: cross_core_reservation_order.facility.to_s, match: :first).click
+      find("a", text: "Report an Issue").click
```

Comment:
maybe put this into a `within` block to make it explicit which order detail we're reporting an issue about